woocommerce/woocommerce
# Storybook: Fix after update to PNPM 7 broke build

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/platform-private/issues/100

### How to test the changes in this Pull Request:

1. Install all the things `pnpm install`
1. Make sure Storybook builds `pnpm --filter=@woocommerce/storybook build-storybook`
2. And confirm the tool is usable`pnpm --filter=@woocommerce/storybook storybook`
3. Make sure nothing else is broken `pnpm build`


### Other information:




### FOR PR REVIEWER ONLY:
